app/src/app/classes/stix/stix-object.ts

```diff
@@ -322,7 +323,7 @@ export abstract class StixObject extends Serializable {
                             }
                         }
                         // check ATT&CK ID and ignore collections
-                        if (this.attackType != "collection" && this.hasOwnProperty("attackID")) {
+                        if (this.hasOwnProperty("supportsAttackID") && this.supportsAttackID == true) {
```

Comment:
Not really worth changing, but `this.hasOwnProperty("supportsAttackID")` I think always resolves to `true` anyway since abstract readonly properties must be assigned in classes extending StixObject.